server/test/controllers/admin/AdminApplicationControllerTest.java

```diff
@@ -363,6 +363,55 @@ public void updateNote_notAdmin() throws Exception {
     assertThat(result.status()).isEqualTo(UNAUTHORIZED);
   }
 
+  @Test
+  public void updateNote_noNote_fails() throws Exception {
+    // Setup.
+    Account adminAccount = resourceCreator.insertAccount();
+    controller = makeNoOpProfileController(Optional.of(adminAccount));
+    Program program = ProgramBuilder.newDraftProgram("test name", "test description").build();
+    Applicant applicant = resourceCreator.insertApplicantWithAccount();
+    Application application =
+        Application.create(applicant, program, LifecycleStage.ACTIVE).setSubmitTimeToNow();
+
+    Request request = addCSRFToken(Helpers.fakeRequest()).build();
+    // Execute.
+    Result result = controller.updateNote(request, program.id, application.id);
+
+    // Verify.
+    assertThat(result.status()).isEqualTo(BAD_REQUEST);
+    assertThat(contentAsString(result)).contains("A note is not present");
+  }
+
+  @Test
+  public void updateNote_succeeds() throws Exception {
```

Comment:
Mind adding a test with a note of the empty string "" succeeding? I think supporting this is definitely desirable since it's effectively "clearing" any existing notes. I just think it'll be helpful to have a test case that confirms explicit support for that value (and that it's not, for example, treated as a bad request).

Reply:
agreed and done!